guides/common/assembly_preparing-environment-for-project-server-installation.adoc

```diff
@@ -0,0 +1,16 @@
+ifdef::context[:parent-context: {context}]
+
+include::modules/con_preparing-environment-for-project-server-installation.adoc[]
+
+include::modules/proc_enabling-client-connections-to-satellite.adoc[leveloffset=+1]
+
+include::modules/proc_verifying-dns-resolution.adoc[leveloffset=+1]
+
+ifdef::parent-context[:context: {parent-context}]
+ifndef::parent-context[:!context:]
```

Comment:
@aneta-petrova @maximiliankolb this is the reassignment statement I am referring to.

Reply:
@aneta-petrova thanks! I tested it locally and looks good. 
I was wondering if this was probably put there for smart-proxy installation, but Smart Proxy Installation guide does not have any modules in common with the Project Installation guides.
Dropping these lines.